internal/operators/odf/validations.go

```diff
@@ -168,16 +168,19 @@ func (o *operator) setStatusInsufficientResources(odfClusterResources *odfCluste
 // count all disks of drive type ssd or hdd
 func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string, mode odfDeploymentMode) (int64, error) {
 	var countDisks int64
```

Comment:
Let's rename to ``countRegularDisks`` or ``countEligibleDisks`` so we'll know it's not the total count of disks

Reply:
so, I want to create a follow up tasks to change this function. For now I created a dirty fix just for the bug.

so this function should return only the amount of eligible disks without an error and move all the error handling to the main / called by function.

it will be more readable and we can have more complex logic that needed to be add to the validation